xmipp

```diff
@@ -554,6 +556,20 @@ def configOpenCV(configDict):
 def getDependenciesInclude():
     return ['../']
 
+def askPath(default='', noAsk=False):
```

Comment:
Personally, I prefer non-negative booleans. I find them easier to work with (consider `noAsk=False` == `ask=True`)